config/configmap/inferenceservice.yaml

```diff
@@ -123,6 +124,10 @@ data:
      # The rest of the fields are used in both authentication methods (IAM Role for Service Account & IAM User Access Key Secret) if a non-empty value is provided.
      credentials: |-
        {
+          # KServe by default gets the secret reference from service account, the configured annotation allows overriding with
```

Comment:
nit: `KServe by default gets the secret reference from service account` if you use storage uri instead of storage spec.